### ISSUE TITLE
Traha Global (2119490)

### ISSUE THREAD

**Issue by Big-FG** [ValveSoftware/Proton#6361]
# Compatibility Report
- Name of the game with compatibility issues: Traha Global
- Steam AppID of the game: 2119490

## System Information
- GPU: RX6800
- Driver/LLVM version: Mesa-Git LLVM 14.0.6
- Kernel version: 6.0.10-272-tkg-cfs
- Link to full system information report: https://gist.github.com/Big-FG/bb721dd489a95036fd57b252bba80eca 
- Proton version: 7.0-4

## I confirm:
- [ X] that I haven't found an existing compatibility report for this game.
- [ X] that I have checked whether there are updates for my system available.

## Symptoms 

Game Largely Works just fine out of the box, the only issue is that the Reflections and Shadows are upside down. I think this means the depth buffer is flipped? but im unsure as to a way to remedy this, turning down the settings works around the issue by eliminating shadows and ini tweaks can disable reflections. 

EDIT: Forcing TAA through ini flips the 3D output upside down entirely. Ive heard of this with Reshade with TAA flipping the buffer.

This does not appear in any of the game footage for players on Windows.

Happens with different proton/mesa version as well as WineD3D in both opengl and vulkan render mode. Also happens with AMDVLK and AMDGPU-Pro

## Reproduction

Install game, set graphics to Ultra.

![shadows](https://user-images.githubusercontent.com/81116028/205460418-81abb77f-85af-4ea9-a09c-e9128c349efb.png)

![reflections](https://user-images.githubusercontent.com/81116028/205460425-b1b6b585-ef86-441d-93c7-e1cb727eb550.png)

Link to PC version Windows gameplay https://www.youtube.com/watch?v=9JxYGjiPYfg

**Comment by Blisto91** [comment]
Thank you for the report. 
I have reproduced the shadow problem and made a issue in upstream dxvk https://github.com/doitsujin/dxvk/issues/3118

**Comment by Big-FG** [comment]
Thanks to @Blisto91 the game is now 100% functional. Turns out the game is stupid when detecting AMD/Nvidia ID in proton and setting it to an Intel ID fixes things

This also fixed weird LOD issues i thought was just the game being a mobile port

**Comment by Big-FG** [comment]
So I thought the SS reflections issue was fixed but it seems that it is not. This i think at this point is an issue on the games part and not something we can fix. You can set the game to have the same reflections setting as on mobile by editing the Engine.ini in 

"path to"/compatdata/2119490/pfx/drive_c/users/steamuser/AppData/Local/RapaNui/Saved/Config/WindowsNoEditor/

then add 

[/Script/Engine.RendererSettings]
r.SSR.Quality=0

This keeps the shiny surfaces but you dont see some of the weird reflections and this is visually the same as it is on Mobile Ultra. I suspect the SSR setting was just turned up/on for PC and not really tested or its another weird proton issue.

**Comment by Blisto91** [comment]
Ye the issues with the reflections or what it is that can also move around with the player camera seems to be a native game issue as it also shows when i tested it on Windows without dxvk

**Comment by Big-FG** [comment]
> Ye the issues with the reflections or what it is that can also move around with the player camera seems to be a native game issue as it also shows when i tested it on Windows without dxvk

Yeah then its just a game bug, It doesnt happen on mobile because they dont have full SSR enabled and its essentially just glossy surfaces on mobile. Luckily setting SSR Quality like previous makes the mobile and PC versions visually identical.
